tests/testthat/test-property.R

```diff
@@ -178,6 +179,7 @@ test_that("properties can be base, S3, S4, R7, or R7 union", {
   expect_snapshot(my_class)
   my_obj <- my_class(
     anything = TRUE,
```

Comment:
if you use `missing_arg()` as an object I guess it probably doesn't print as missing?

so maybe for this example you just wouldn't supply that argument, and it would still be captured in the snapshot?